### ISSUE TITLE
Scale and Settings: 'Learn more' link is not adapting the high contrast mode after applying high contrast theme.

### ISSUE THREAD

**Issue by ato9000** [Azure/cosmos-explorer#1055]
https://msdata.visualstudio.com/CosmosDB/_workitems/edit/1245554

**Comment by ato9000** [comment]
Note: User credentials should NOT be included in the bug. 

Pre-Requisite:
Select "Core(SQL)" in API combo box while creating the Cosmos DB.
Ease of access high contrast settings > Turn on High Contrast > High Contrast Black/High Contrast White

Repro Steps:
Open URL: https://ms.portal.azure.com/ and login using valid credentials.
Navigate to "Azure Cosmos DB " from left pane and select it.
Navigate to any of the Azure Cosmos DB which is already created and select it.
Navigate to "Data Explorer" from the left and select it.
Navigate through all the controls under SQL API Section.
Create 'Sample Database'. Expand this section and then expand 'Persons' tree option.
Navigate through all the controls under SQL API Section and select "Scale & Settings".
Navigate to scale tab section controls and select 'Auto scale' radio button.
Enable high contrast black mode.
Verify the issue.

Actual Result:
'Learn more' link is not adapting the high contrast black mode after applying high contrast theme.
It is visible in 'Blue' color at high contrast black mode.

Expected Result:
'Learn more' link should adapt the high contrast black mode after applying high contrast theme.
It should be visible in 'yellow' color in high contrast black mode and 'Purple' color in high contrast white mode.

Note:
Similar issue exists under high contrast white mode.

![image](https://user-images.githubusercontent.com/8519407/132715625-67be5450-a3b1-4797-bfcc-0359fe183f0c.png)

![image](https://user-images.githubusercontent.com/8519407/132715652-2950105a-c755-4991-887f-2ca11ed3f09f.png)
